content/doc/reference/reference-environment-variables.md

```diff
@@ -234,7 +234,7 @@ If `TAILSCALE_LOGIN_SERVER` is provided, the agent will be configured to reach a
 |[`CC_PHP_ASYNC_APP_BUCKET`](/doc/applications/php/#speed-up-or-disable-the-session-fs-bucket "Speed up or disable the session on FS Bucket") | Mount the default app FS bucket asynchronously. If set, should have value `async` |  |
 |[`CC_PHP_DEV_DEPENDENCIES`](/doc/applications/php/#development-dependencies "Development dependencies") | Control if development dependencies are installed or not. Values are either `install` or `ignore` |  |
 |[`CC_PHP_DISABLE_APP_BUCKET`](/doc/applications/php/#speed-up-or-disable-the-session-fs-bucket "Speed up or disable the session on FS Bucket") | Disable entirely the app FS Bucket. Values are either `true`, `yes` or `disable` |  |
-|`CC_PHP_VERSION` | Choose your PHP version between 5.6, 7.2, 7.3, 7.4, 8.0, 8.1 and 8.2 | 8 |
+|`CC_PHP_VERSION` | Choose your PHP version between 5.6, 7.2, 7.3, 7.4, 8.0, 8.1, 8.2 and 8.3 | 8 |
```

Comment:
Maybe we can link to the automated part of PHP doc to avoid duplicate and misalignment? https://developers.clever-cloud.com/doc/applications/php/#choose-your-php-version

Reply:
Good idea, comment converted in issue.